.rubocop_todo.yml

```diff
@@ -126,7 +117,6 @@ RSpec/MessageSpies:
     - 'spec/preservation_ingest/complete_ingest_spec.rb'
     - 'spec/preservation_ingest/transfer_object_spec.rb'
     - 'spec/preservation_ingest/update_moab_spec.rb'
-    - 'spec/preservation_ingest/verify_apo_spec.rb'
```

Comment:
haha - did you hand edit this?  the counts didn't go down.  And it doesn't matter.

Reply:
yes, guilty

a full regen was a lot of changes